Controllers/AuthenticationController.py

```diff
@@ -22,4 +22,15 @@ def login():
 Request params: first name, last name, email (will be used as username), password
 Return: Success or failure codes    
 """
-
+@auth.route("/signup", methods=["GET","POST"])
+def signup():
+    if request.method == "POST":
+        user = {"firstName": request.form["firstName"], 
+                "lastName": request.form["lastName"], 
+                "email": request.form["email"],
+                "password": request.form["password"]}
+        success,error = AuthenticationService.signup(user)  # Authenticate signup route
```

Comment:
Is this valid python ?

Reply:
Yes, AuthenticationService.signup returns two args, so this would be fine